gslab_scons/log.py

```diff
@@ -78,7 +82,11 @@ def collect_builder_logs(parent_dir):
                 with open(f_full, 'rU') as g:
                     s = g.readlines()[1] # skip a line
                     s = s[s.find('{')+1: s.find('}')] # find {} time identifier 
-                    builder_log_end_time = datetime.strptime(s, "%Y-%m-%d %H:%M:%S")
-                builder_log_collect[f_full] = builder_log_end_time
+                    try:
+                        builder_log_end_time = datetime.strptime(s, "%Y-%m-%d %H:%M:%S")
+                    except ValueError: # if the code breaks, there's no time identifier
+                        beginning_of_time    = datetime.min
+                        builder_log_end_time = beginning_of_time
+                builder_log_collect[f_full]  = builder_log_end_time
```

Comment:
Maybe I'm not seeing it, but is this code too redundant? Can't we equivalently write

```python 
 builder_log_collect[f_full] = datetime.min
```

Also, what's the rationale for using this specific failure time?